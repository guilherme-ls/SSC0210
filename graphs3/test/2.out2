amor total
amor total
amor total
[1,1][2,1][3,2]
amor total
[1,3][2,1]
amor total
[1,1][2,4]
amor total
[1,4][2,1]
amor total
amor total
amor total
[1,1][2,3][3,2]
amor total
amor total
amor total
[1,5][2,1]
[1,3][2,3]
[1,3][2,3]
[1,2][2,2][3,2]
[1,1][2,1][3,2][4,1][5,1]
amor total
amor total
amor total
[1,1][2,2][3,2][4,2]
amor total
amor total
amor total
amor total
amor total
amor total
amor total
amor total
amor total
amor total
amor total
amor total
[1,4][2,4]
[1,1][2,4][3,3]
[1,2][2,2][3,2][4,2]
[1,2][2,2][3,2][4,1][5,1]
[1,4][2,4]
[1,4][2,3][3,1]
amor total
amor total
amor total
amor total
amor total
[1,1][2,9]
amor total
amor total
amor total
amor total
amor total
amor total
amor total
amor total
[1,5][2,5]
[1,1][2,2][3,1][4,1][5,4][6,1]
[1,5][2,5]
[1,1][2,1][3,1][4,2][5,5]
[1,2][2,2][3,2][4,2][5,2]
[1,1][2,1][3,2][4,2][5,2][6,2]
[1,5][2,5]
[1,5][2,5]
amor total
amor total
amor total
amor total
amor total
amor total
amor total
[1,1][2,3][3,1][4,1][5,2][6,2][7,1][8,1][9,2][10,3][11,2][12,3][13,8][14,1][15,1][16,2][17,5][18,1]
amor total
[1,1][2,10]
amor total
[1,1][2,1][3,1][4,3][5,4][6,2][7,1][8,1][9,1]
amor total
[1,1][2,1][3,1][4,1][5,2][6,5][7,4][8,2][9,1][10,1][11,7][12,5]
amor total
[1,3][2,3][3,1][4,1][5,1][6,4][7,2][8,4][9,1]
amor total
[1,2][2,3][3,1][4,1][5,4][6,2][7,2][8,1][9,2][10,3][11,1][12,3][13,3][14,1][15,1][16,1][17,2][18,2][19,1][20,2][21,2]
amor total
[1,1][2,2][3,1][4,2][5,2][6,1][7,1][8,4][9,2][10,2][11,2][12,3][13,1][14,2][15,1][16,4][17,1][18,1][19,6]
amor total
[1,1][2,1][3,2][4,3][5,1][6,1][7,1]
amor total
[1,3][2,1]
amor total
[1,2][2,1][3,2]
amor total
[1,3][2,3][3,5][4,2][5,3][6,1][7,2][8,1][9,2][10,2][11,8][12,2][13,2][14,3][15,1]
amor total
[1,2][2,1][3,4][4,1][5,3][6,2][7,2][8,2][9,2][10,1][11,1][12,5][13,1][14,4][15,4][16,1][17,2][18,1]
amor total
[1,1][2,3][3,3][4,1]
amor total
[1,1][2,2][3,1][4,3]
amor total
amor total
amor total
[1,5][2,5]
amor total
[1,4][2,5]
amor total
[1,4][2,4]
amor total
[1,12][2,1][3,1][4,1][5,1][6,10]
amor total
[1,9][2,3][3,10]
amor total
[1,21][2,19]
amor total
amor total
amor total
[1,2][2,3][3,1][4,1]
amor total
[1,3][2,3]
amor total
[1,4][2,1][3,1][4,4]
amor total
[1,4][2,5]
amor total
[1,4][2,4]
amor total
[1,20][2,20]
amor total
[1,1][2,3][3,2][4,1]
amor total
[1,1][2,3][3,10]
amor total
amor total
amor total
[1,1][2,2][3,10]
amor total
amor total
amor total
[1,3][2,3]
amor total
[1,10][2,2][3,1]
amor total
[1,1][2,10]
amor total
amor total
amor total
[1,2][2,37]
amor total
amor total
amor total
[1,1][2,17]
amor total
amor total
amor total
amor total
amor total
amor total
amor total
[1,1][2,3][3,1]
amor total
[1,4][2,1]
amor total
amor total
[1,5][2,5][3,5]
[1,4][2,1][3,4][4,1][5,1][6,3][7,1]
amor total
amor total
amor total
[1,2][2,2][3,22]
amor total
[1,18][2,1][3,4][4,1]
amor total
amor total
amor total
[1,1][2,4][3,1][4,21][5,1][6,6][7,6]
amor total
[1,3][2,1][3,2]
amor total
[1,1][2,2][3,2][4,1][5,1][6,1][7,12][8,3][9,2][10,1]
amor total
[1,2][2,22]
amor total
[1,1][2,1][3,14][4,6][5,2][6,4][7,12]
amor total
amor total
amor total
amor total
amor total
[1,1][2,3][3,1]
amor total
amor total
amor total
amor total
amor total
amor total
amor total
amor total
amor total
[1,1][2,38][3,1]
amor total
amor total
amor total
amor total
amor total
amor total
amor total
amor total
amor total
[1,2][2,26]
amor total
[1,1][2,19]
amor total
amor total
amor total
[1,4][2,2][3,23][4,1]
amor total
amor total
amor total
amor total
amor total
[1,1][2,1][3,1][4,20][5,1][6,2]
amor total
[1,1][2,15][3,2][4,1][5,1]
amor total
amor total
amor total
[1,1][2,2][3,7][4,1][5,1]
amor total
amor total
amor total
amor total
amor total
amor total
amor total
amor total
amor total
amor total
amor total
amor total
amor total
[1,1][2,7][3,2]
amor total
[1,1][2,15]
amor total
amor total
amor total
amor total
amor total
[1,1][2,2][3,1][4,1][5,2][6,3]
amor total
amor total
amor total
amor total
amor total
[1,1][2,1][3,1][4,1][5,6][6,1][7,3][8,2][9,3][10,1][11,2][12,2][13,1][14,2][15,1][16,5][17,1][18,4][19,2]
amor total
amor total
amor total
amor total
[1,1][2,9]
[1,1][2,3][3,1][4,5]
amor total
amor total
[1,1][2,2][3,1][4,1][5,1][6,1][7,1][8,1][9,1][10,1][11,2][12,1][13,1][14,2][15,1][16,1][17,2][18,1][19,1][20,1][21,1][22,3][23,2][24,1][25,1]
[1,1][2,1][3,2][4,1][5,1][6,1][7,1][8,1][9,1][10,1][11,1][12,1][13,1][14,1][15,1][16,1][17,1][18,1][19,1][20,2][21,1][22,1][23,1][24,1][25,2][26,2][27,1][28,1]
[1,1][2,1][3,1][4,1][5,1][6,1][7,1][8,2][9,2][10,1][11,28]
[1,1][2,1][3,1][4,1][5,1][6,1][7,1][8,1][9,1][10,1][11,1][12,1][13,1][14,1][15,2][16,1][17,1][18,7][19,1][20,1][21,1][22,1][23,7][24,4]
amor total
amor total
amor total
amor total
amor total
amor total
amor total
amor total
amor total
amor total
amor total
amor total
amor total
[1,2][2,2][3,2][4,2][5,2][6,3][7,3][8,1][9,2]
amor total
[1,2][2,2][3,2][4,2][5,2][6,3][7,1][8,2][9,2][10,2][11,1][12,2]
amor total
[1,1][2,1][3,4][4,1][5,1][6,4][7,5][8,1]
amor total
[1,1][2,1][3,4][4,1][5,2][6,3][7,2]
amor total
amor total
amor total
[1,1][2,1][3,2][4,1]
amor total
[1,1][2,6][3,1][4,1][5,1]
amor total
[1,1][2,1][3,1][4,1][5,1][6,1][7,1][8,16][9,1][10,1][11,1][12,1][13,1][14,1][15,1][16,1][17,1][18,1][19,1][20,1][21,1][22,1][23,1][24,1][25,1]
amor total
[1,1][2,2][3,1][4,2]
amor total
amor total
amor total
amor total
amor total
[1,9][2,1]
amor total
[1,9][2,1]
amor total
[1,1][2,1][3,5][4,1][5,2]
amor total
[1,1][2,1][3,1][4,1][5,1][6,1][7,1][8,1][9,1][10,1][11,1][12,1][13,1][14,1][15,3][16,1][17,13][18,3][19,1][20,1][21,1][22,2][23,1]
amor total
[1,1][2,33][3,1][4,1][5,1][6,1][7,2]
amor total
amor total
amor total
amor total
amor total
amor total
amor total
[1,3][2,1][3,5][4,1]
amor total
amor total
[1,3][2,1]
[1,1][2,1][3,1][4,1]
amor total
