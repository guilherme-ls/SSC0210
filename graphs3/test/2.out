amor total
amor total
amor total
[1,1][2,1][3,2]
amor total
[1,1][2,3]
amor total
[1,4][2,1]
amor total
[1,4][2,1]
amor total
amor total
amor total
[1,3][2,2][3,1]
amor total
amor total
amor total
[1,5][2,1]
[1,3][2,3]
[1,3][2,3]
[1,2][2,2][3,2]
[1,1][2,1][3,2][4,1][5,1]
amor total
amor total
amor total
[1,1][2,2][3,2][4,2]
amor total
amor total
amor total
amor total
amor total
amor total
amor total
amor total
amor total
amor total
amor total
amor total
[1,4][2,4]
[1,3][2,4][3,1]
[1,2][2,2][3,2][4,2]
[1,1][2,2][3,2][4,2][5,1]
[1,4][2,4]
[1,3][2,4][3,1]
amor total
amor total
amor total
amor total
amor total
[1,9][2,1]
amor total
amor total
amor total
amor total
amor total
amor total
amor total
amor total
[1,5][2,5]
[1,4][2,1][3,1][4,2][5,1][6,1]
[1,5][2,5]
[1,5][2,1][3,2][4,1][5,1]
[1,2][2,2][3,2][4,2][5,2]
[1,2][2,2][3,2][4,2][5,1][6,1]
[1,5][2,5]
[1,5][2,5]
amor total
amor total
amor total
amor total
amor total
amor total
amor total
[1,1][2,5][3,1][4,8][5,1][6,3][7,2][8,2][9,1][10,1][11,2][12,3][13,2][14,1][15,1][16,1][17,3][18,2]
amor total
[1,10][2,1]
amor total
[1,4][2,3][3,2][4,1][5,1][6,1][7,1][8,1][9,1]
amor total
[1,4][2,5][3,5][4,7][5,2][6,2][7,1][8,1][9,1][10,1][11,1][12,1]
amor total
[1,4][2,1][3,2][4,4][5,1][6,1][7,1][8,3][9,3]
amor total
[1,2][2,2][3,2][4,1][5,2][6,1][7,1][8,1][9,3][10,3][11,1][12,1][13,2][14,3][15,2][16,2][17,4][18,1][19,1][20,2][21,3]
amor total
[1,1][2,6][3,1][4,1][5,4][6,1][7,2][8,2][9,2][10,2][11,3][12,1][13,4][14,1][15,1][16,2][17,2][18,1][19,2]
amor total
[1,1][2,3][3,1][4,1][5,1][6,1][7,2]
amor total
[1,3][2,1]
amor total
[1,2][2,1][3,2]
amor total
[1,3][2,1][3,2][4,8][5,2][6,2][7,2][8,1][9,1][10,2][11,5][12,2][13,3][14,3][15,3]
amor total
[1,2][2,1][3,4][4,1][5,4][6,1][7,5][8,1][9,1][10,2][11,2][12,2][13,2][14,1][15,4][16,1][17,3][18,2]
amor total
[1,3][2,1][3,3][4,1]
amor total
[1,3][2,1][3,2][4,1]
amor total
amor total
amor total
[1,5][2,5]
amor total
[1,5][2,4]
amor total
[1,4][2,4]
amor total
[1,10][2,1][3,1][4,1][5,1][6,12]
amor total
[1,10][2,3][3,9]
amor total
[1,21][2,19]
amor total
amor total
amor total
[1,2][2,1][3,1][4,3]
amor total
[1,3][2,3]
amor total
[1,4][2,4][3,1][4,1]
amor total
[1,4][2,5]
amor total
[1,4][2,4]
amor total
[1,20][2,20]
amor total
[1,3][2,2][3,1][4,1]
amor total
[1,10][2,1][3,3]
amor total
amor total
amor total
[1,10][2,2][3,1]
amor total
amor total
amor total
[1,3][2,3]
amor total
[1,1][2,2][3,10]
amor total
[1,1][2,10]
amor total
amor total
amor total
[1,2][2,37]
amor total
amor total
amor total
[1,17][2,1]
amor total
amor total
amor total
amor total
amor total
amor total
amor total
[1,1][2,3][3,1]
amor total
[1,4][2,1]
amor total
amor total
[1,5][2,5][3,5]
[1,1][2,3][3,1][4,1][5,4][6,4][7,1]
amor total
amor total
amor total
[1,22][2,2][3,2]
amor total
[1,4][2,18][3,1][4,1]
amor total
amor total
amor total
[1,21][2,6][3,6][4,1][5,4][6,1][7,1]
amor total
[1,2][2,3][3,1]
amor total
[1,2][2,1][3,1][4,12][5,3][6,1][7,1][8,2][9,2][10,1]
amor total
[1,22][2,2]
amor total
[1,14][2,6][3,2][4,4][5,12][6,1][7,1]
amor total
amor total
amor total
amor total
amor total
[1,3][2,1][3,1]
amor total
amor total
amor total
amor total
amor total
amor total
amor total
amor total
amor total
[1,38][2,1][3,1]
amor total
amor total
amor total
amor total
amor total
amor total
amor total
amor total
amor total
[1,26][2,2]
amor total
[1,19][2,1]
amor total
amor total
amor total
[1,23][2,2][3,4][4,1]
amor total
amor total
amor total
amor total
amor total
[1,20][2,2][3,1][4,1][5,1][6,1]
amor total
[1,15][2,2][3,1][4,1][5,1]
amor total
amor total
amor total
[1,7][2,2][3,1][4,1][5,1]
amor total
amor total
amor total
amor total
amor total
amor total
amor total
amor total
amor total
amor total
amor total
amor total
amor total
[1,7][2,2][3,1]
amor total
[1,15][2,1]
amor total
amor total
amor total
amor total
amor total
[1,2][2,3][3,1][4,2][5,1][6,1]
amor total
amor total
amor total
amor total
amor total
[1,1][2,2][3,2][4,4][5,5][6,3][7,2][8,6][9,3][10,1][11,1][12,1][13,1][14,2][15,1][16,1][17,2][18,1][19,1]
amor total
amor total
amor total
amor total
[1,9][2,1]
[1,1][2,3][3,5][4,1]
amor total
amor total
[1,1][2,1][3,2][4,3][5,1][6,1][7,1][8,1][9,2][10,1][11,1][12,2][13,1][14,1][15,2][16,1][17,1][18,1][19,1][20,1][21,1][22,1][23,1][24,2][25,1]
[1,1][2,1][3,2][4,2][5,1][6,1][7,1][8,1][9,2][10,1][11,1][12,1][13,1][14,1][15,1][16,1][17,1][18,1][19,1][20,1][21,1][22,1][23,1][24,1][25,1][26,2][27,1][28,1]
[1,28][2,1][3,2][4,2][5,1][6,1][7,1][8,1][9,1][10,1][11,1]
[1,7][2,7][3,1][4,1][5,4][6,1][7,1][8,1][9,2][10,1][11,1][12,1][13,1][14,1][15,1][16,1][17,1][18,1][19,1][20,1][21,1][22,1][23,1][24,1]
amor total
amor total
amor total
amor total
amor total
amor total
amor total
amor total
amor total
amor total
amor total
amor total
amor total
[1,2][2,1][3,2][4,3][5,3][6,2][7,2][8,2][9,2]
amor total
[1,2][2,2][3,2][4,2][5,3][6,1][7,2][8,2][9,2][10,1][11,2][12,2]
amor total
[1,5][2,1][3,1][4,4][5,1][6,4][7,1][8,1]
amor total
[1,2][2,3][3,2][4,4][5,1][6,1][7,1]
amor total
amor total
amor total
[1,2][2,1][3,1][4,1]
amor total
[1,6][2,1][3,1][4,1][5,1]
amor total
[1,16][2,1][3,1][4,1][5,1][6,1][7,1][8,1][9,1][10,1][11,1][12,1][13,1][14,1][15,1][16,1][17,1][18,1][19,1][20,1][21,1][22,1][23,1][24,1][25,1]
amor total
[1,1][2,2][3,2][4,1]
amor total
amor total
amor total
amor total
amor total
[1,9][2,1]
amor total
[1,9][2,1]
amor total
[1,5][2,1][3,2][4,1][5,1]
amor total
[1,13][2,1][3,2][4,3][5,3][6,1][7,1][8,1][9,1][10,1][11,1][12,1][13,1][14,1][15,1][16,1][17,1][18,1][19,1][20,1][21,1][22,1][23,1]
amor total
[1,33][2,1][3,1][4,1][5,2][6,1][7,1]
amor total
amor total
amor total
amor total
amor total
amor total
amor total
[1,5][2,1][3,3][4,1]
amor total
amor total
[1,1][2,3]
[1,1][2,1][3,1][4,1]
amor total
